### ISSUE TITLE
Make the email format / order of items match the CID dashboard appearance

### ISSUE THREAD

**Issue by njosefbeck** [FHDevelopment/fhCID#5]
Email:
![image](https://cloud.githubusercontent.com/assets/9508002/12247049/b8038fa2-b877-11e5-8dfe-5c01587812ee.png)

CID Dashboard:
![image](https://cloud.githubusercontent.com/assets/9508002/12247044/af77e34c-b877-11e5-8366-b0b01f83fafb.png)

**Comment by Goswamiv** [comment]
@donmox @njosefbeck @walshmp 
On CID dashboard, it is pulling the data ordered by 'priority' attribute which is still sitting in XMLs base.
The EmailSender doesnt talk with XMLs. I think we can make it pull data from xmls but that is a big design issue and not the right way to do. Basically, the xmls and db frameworks cant go together really well and it will keep becoming bigger issue as we keep adding features.

I will invest some time to figure out how much dev effort we need to move everythign off XML.

**Comment by Goswamiv** [comment]
`   var companyconfigs = (from configs in _DashBoardConfig.Items
                                  where configs.Group.GroupId == companygroup.Id
                                  //orderby configs.Display.Priority ascending
                                  orderby configs.SearchDescription descending`

Looks like the code is already commented out for priority order. may be it was never pushed to PROD. So based on the code changes, this feature where it orders by priority is an obsolete feature.

**Comment by njosefbeck** [comment]
So does that mean that by just un-commenting out, it should order the items in the email by priority?

**Comment by Goswamiv** [comment]
I am not sure why would we remove it when we wanted this feature.
Anyways, I am checking to see if we can put this feature back as I think according to @donmox - we want this feature.

**Comment by njosefbeck** [comment]
Yeah, it seems like we want the feature, so I also don't know why it was commented out in the first place...

**Comment by Goswamiv** [comment]
@njosefbeck 
I made the change somewhere in monolith and deployed the email job. lets see if it works.
Let me know if you receive the email which displays in asc order or priority.

**Comment by njosefbeck** [comment]
@Goswamiv Neither Don nor I have gotten a new Ross email, though new items for today (Jan 14) are showing in the CID dashboard:

https://cid.fhtrue.com/SearchGroup.aspx?ClientGuid=85bce403-3630-497b-b21c-bed385894dc6
(The screenshot is small, but there are items showing Jan 14 date.)
![screen shot 2016-01-14 at 8 26 57 am](https://cloud.githubusercontent.com/assets/9508002/12326829/a0fe265a-ba98-11e5-9241-8aa9c657cd2a.png)

**Comment by njosefbeck** [comment]
Don did get an email about Omnicom last night from CID with dates from yesterday:
![screen shot 2016-01-14 at 8 32 21 am](https://cloud.githubusercontent.com/assets/9508002/12326948/62fed11e-ba99-11e5-8b12-4f638b9adab3.png)

**Comment by Goswamiv** [comment]
We have Daily CID job which runs 7:50 AM UTC  (1:50 AM CST) every day which sends daily digest.
"As it comes" notifications are being sent every hour for any newitems.

I will have to check with Mike few cases 
e.g.  What happens to newitems notifications if it was already sent as part of daily/weekely notifications
